aiosql/adapters/__init__.py

```diff
@@ -0,0 +1,19 @@
+# standard adapters
+from .pyformat import PyFormatAdapter
+from .generic import GenericAdapter
+from .sqlite3 import SQLite3Adapter
+from .psycopg import PsycoPGAdapter
+
+# async adapters
+from .aiosqlite import AioSQLiteAdapter
+from .asyncpg import AsyncPGAdapter
+
+# silence flake8 F401 warning:
+_ALL = [
```

Comment:
Why not just use `__all__`?

Reply:
There is a type error if I do that because `mypy` expects `__all__` to be a `Sequence`, not a `List`...